autoload/LanguageClient.vim

```diff
@@ -451,16 +451,15 @@ function! LanguageClient#Notify(method, params) abort
 endfunction
 
 function! LanguageClient#textDocument_hover(...) abort
-    let l:callback = get(a:000, 1, v:null)
```

Comment:
Instead of inline all the operations of getting the function, can you instead change the variable to upper case? i.e., `l:Callback` should work.

Reply:
Good catch. Have reverted all the `l:callback` to `:l:Callback`.